specs/Energy/ThreePhaseMultiCircuitAcMeasurement/example-normalized.json

```diff
@@ -0,0 +1,96 @@
+{
+    "id": "ThreePhaseAcMultiCircuitMeasurement:LV3_Ventilation",
+    "type": "ThreePhaseAcMultiCircuitMeasurement",
+    "current": {
+        "value": [
+            33.55, 
```

Comment:
instead of providing a value for each circuit why don't you just create a new entity per circuit?

Reply:
@jmcanterafonseca We gave it a try, but we find `circuit` to be very tightly coupled with the device as well as other circuits on the same device. And so we don't see a benefit in breaking it down into a new entity.

I've come up with an alternative schema that one might find easier to use : https://github.com/LincWorld/dataModels/commit/4209aff7f5ce0c4b4d6c54934ace7c7b64e598f3
Let me know if you prefer it. I'll send a pull request for that.

Reply:
@brahma-dev your new proposal sounds good to me. Please re-do this PR and then I will review